.github/workflows/test.yml

```diff
@@ -55,10 +55,75 @@ jobs:
             *.cache-to=type=gha,scope=dev${{ matrix.mode }},mode=max
             *.output=type=cacheonly
 
+  validate:
+    runs-on: ubuntu-20.04
+    needs:
+      - build-dev
+    strategy:
+      fail-fast: true
+      matrix:
```

Comment:
Just a question; is there a way to influence naming of these jobs? Some of them are now very long (and show up truncated), and because they're generated, their name may change if (e.g.) new test-suites are added;

```
- test / validate (pkg-imports) (pull_request) Successful in 1m
- test / validate (yamllint) (pull_request) Successful in 1m
- test / validate (swagger) (pull_request) Successful in 1m
- windows-2022 / run / integration-test (builtin, DockerCLICpSuite|DockerCLICreateSuite|DockerCLIEventSuite|DockerCLIExe... (pull_request) Successful in 16m
- windows-2022 / run / integration-test (builtin, DockerCLIPortSuite|DockerCLIProxySuite|DockerCLIPruneSuite|DockerCLIPs... (pull_request) Successful in 10m
```

If we would be able to give them our own name, we could keep the name shorter, and prevent the names from changing (making it easier to mark them as "required"), e.g.

```
- test / validate / pkg-imports (or just validate/pkg-imports)
- test / validate / yamllint
- test / validate / swagger
- windows-2022 / run / integration-test / group-1
- windows-2022 / run / integration-test / group-2
```

Reply:
Unfortunately we can't atm, it takes what is set to the matrix and we can't set objects to a dynamic matrix afaik to have smth like:

```
- name: group1
  value: DockerCLICpSuite|DockerCLICreateSuite|DockerCLIEventSuite|DockerCLIExe...
```